serverless/develop/serverless-kafka-developer.adoc

```diff
@@ -34,8 +39,25 @@ include::modules/specifying-sink-flag-kn.adoc[leveloffset=+3]
 // YAML
 include::modules/serverless-kafka-source-yaml.adoc[leveloffset=+2]
 
+[id="serverless-kafka-developer-broker"]
+== Kafka broker
+
+:FeatureName: Kafka broker
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka broker as the default broker type, xref:../../serverless/develop/serverless-using-brokers.adoc#serverless-using-brokers-creating-brokers[creating a broker by using the default settings] creates a Kafka-based `Broker` object instead of a generic `Broker` object.
```

Comment:
> Kafka-based `Broker` object instead of a generic `Broker` object.

I think this is OK-ish But one more comment here:

The `Broker` API is a generic API, that has different implementations. We used to ship just the "Channel-based" broker. We now have a Kafka-based broker. Both have same API (`Broker` CRD). On the runtime (or on the cluster) you differentiate which one you use by applying an annotation.

Currently the "Channel-based" broker is configured as default, so you do not need to provide the annotation there.


With that in mind, would it make sense to say:

> creates a Kafka-based `Broker` object instead of a Channel-based `Broker` object?   

Especially with good introduction that @devguyio  suggested for more clarification.

Reply:
Made some updates, removed entirely the bit about the generic broker and just emphasize the Kafka broker